core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -87,6 +94,17 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
       case str: String if str.nonEmpty => str.split(",").map(s => KafkaPrincipal.fromString(s.trim)).toSet
     }.getOrElse(Set.empty[KafkaPrincipal])
 
+    def getInstanceForConfig(configName: String): Option[Nothing] = {
+      configs.get(configName).collect {
+        case str: String if str.nonEmpty => CoreUtils.createObject(str)
+      }
+    }
+    principalToLocalPlugin = getInstanceForConfig(SimpleAclAuthorizer.PrincipalToLocalProp)
```

Comment:
Note that we currently have `SaslConfigs.SASL_KERBEROS_PRINCIPAL_TO_LOCAL_RULES` and there is a `KerberosShortNamer` class. It may be worth thinking about how that fits with this.